api/src/main/java/com/github/streamshub/console/api/ClientFactory.java

```diff
@@ -438,6 +451,7 @@ Map<String, Object> requiredConsumerConfig() {
         configs.put(ConsumerConfig.SESSION_TIMEOUT_MS_CONFIG, 50_000);
         configs.put(ConsumerConfig.AUTO_OFFSET_RESET_CONFIG, "earliest");
         configs.put(ConsumerConfig.DEFAULT_API_TIMEOUT_MS_CONFIG, 5000);
+        configs.put(SerdeConfig.ENABLE_HEADERS, "true");
```

Comment:
https://github.com/Apicurio/apicurio-registry/pull/5149 disables headers by default. Is Console aware of that? Is Console in sympathy with that?

Reply:
The code will currently check the headers first, then fall back to the embedded ID in the message for de-serialization. 

This configuration is more significant for the message producer, which isn't yet part of the UI, so the risk is probably low. I would say we can remove this option here and defer how to handle headers vs. embedded schema identifiers until we add the ability to produce messages to the UI. There may end up being options in the UI to choose where to put the ID, which schema to use, etc.

Reply:
After further review, I think we should keep this option enabled. Without it, deserialization will not check the headers at all, which limits the console's ability to decode records written by an application that does use the headers.